### PR TITLE
rabbit boots pr

### PR DESCRIPTION
**Pull request by miasabyss**
### **OwO**

### CODE REVIEW

File: Season Data/data/tns/functions/is_wearing_rabbit_boots.mcfunction
Comment (TheDeathlyCow) [inline Season Data/data/tns/functions/is_wearing_rabbit_boots.mcfunction:1]: rename function to `tick_rabbit_boots` - to follow its purpose more clearly

File: Season Data/data/tns/predicates/is_wearing_rabbit_boots.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "entity": "this",
+    "predicate": {
+      "type": "minecraft:player",
+      "equipment": {
+        "feet": {
+          "nbt": "{is_rabbit_boots:1b}"
+        }
+      }
+    }
+  }
```
Comment (TheDeathlyCow) [inline Season Data/data/tns/predicates/is_wearing_rabbit_boots.json:10]: indentation seems off here, reformat this file

**Review by TheDeathlyCow — CHANGES_REQUESTED** [review]
looks good - just a couple small refactors needed

**Review by miasabyss — COMMENTED** [review]
i think thats? good? idk i think the indentation might still be fucked but it should work

**Review by TheDeathlyCow — APPROVED** [review]

**Pull request merged**